### DISCUSSION TITLE
How to get the default background color?

### DISCUSSION THREAD

**Discussion by akazwz** [chakra-ui/chakra-ui#5382] [Q&A]
How to get the default background color? The light mode is white and the dark mode is grey.800, but I want to get this in react

**Comment by fredxwin** [comment]
```
import { mode } from "@chakra-ui/theme-tools"

export const customTheme = extendTheme({
  styles: {
    global: (props) => ({
      "html, body": {
        background: mode("white", "green")(props),  //mode(light mode color, dark mode color)
      },
    }),
  },
})
```

**Reply by Thebigbignooby** [reply]
this is *setting* the background color. OP asked how to *get* the background color

**Comment by MTraveller** [comment]
You can get the current colors from useTheme()

ex. to get the bg color, simple extract it from:

```
const theme = useTheme()
console.log(theme.__cssMap['colors.chakra-body-bg'].value)
```

Cheers :)

**Reply by Philzen** [reply]
Has this mapping just recently been added? In v1.8.8 i cannot find that value. I tried useTheme and want to access the currently rendered `styles.global.body.bg`, however `theme` only contains `styles.global().body.bg` (the unresolved prop function).

**Reply by tombohub** [reply]
this gives value: `var(--chakra-colors-chakra-body-bg)` . How can we get hex color?
